package.json

```diff
@@ -0,0 +1,58 @@
+{
+  "name": "myarcade-backend",
+  "version": "1.0.0",
+  "description": "This is an API for a tenant management system for a shopping mall or arcade",
+  "main": "index.js",
+  "scripts": {
+    "build": "babel api --out-dir dist",
```

Comment:
I don't see a test script, are we not going to write tests?

Reply:
I saw a story for it. So the setup for jest will be done later